src/test/unit/helpers/listHelper.test.ts

```diff
@@ -1,148 +0,0 @@
-import sinon from 'sinon';
-import {
-    formatMetaDataListType,
-    hearingHasParty,
-    isOneOfValidListTypes,
-    isValidList,
-    isValidListType,
-    missingListType,
-    addListDetailsToArray
-} from '../../../main/helpers/listHelper';
-import { HttpStatusCode } from 'axios';
-import fs from 'fs';
-import path from 'path';
-import { PublicationService } from '../../../main/service/PublicationService';
-
-const mockArtefact = {
-    listType: 'CIVIL_DAILY_CAUSE_LIST',
-    listTypeName: 'Civil Daily Cause List',
-    contentDate: '2022-03-24T07:36:35',
-    locationId: '5',
-    artefactId: 'valid-artefact',
-    dateRange: 'Invalid DateTime to Invalid DateTime',
-    contDate: '24 Mar 2022'
-};
-
-sinon.stub(PublicationService.prototype, 'getIndividualPublicationMetadata').resolves(mockArtefact);
-
-describe('List helper', () => {
-    describe('hearing has party', () => {
-        it('Hearing should have party', () => {
-            const rawData = fs.readFileSync(
-                path.resolve(__dirname, '../mocks/hearingparty/crownDailyList.json'),
```

Comment:
This test us still required. Just change it to use another list which still has hearing party